src/main/java/at/bitfire/vcard4android/contactrow/EventBuilder.kt

```diff
@@ -40,10 +40,7 @@ class EventBuilder(dataRowUri: Uri, rawContactId: Long?, contact: Contact, readO
             return null
 
         val dateStr: String = when {
-            dateOrTime.date != null -> {
-                val format = SimpleDateFormat("yyyy-MM-dd", Locale.ROOT)
-                format.format(dateOrTime.date)
-            }
+            dateOrTime.date != null -> dateOrTime.date.toString()
```

Comment:
Are we sure that `toString()` does the right thing, for instance when `dateOrTime` is an `Instant` or a `LocalDateTime` (like the documentation says)?

If it's only used by us and clear that it must be a date (and not for instance an `Instant` or `LocalDateTime`), this should be documented in KDoc and be checked explicitly.

Reply:
I've replaced all the `toString` calls to `DateTimeFormatter`, which replaces `SimpleDateFormat`. Now it should be more trust-worthy